pkgs/games/nsnake/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, pkgs, lib }:
+
+stdenv.mkDerivation rec {
+  pname = "nsnake";
+  version = "3.0.1";
+
+  src = fetchFromGitHub {
+    owner = "alexdantas";
+    repo = "nSnake";
+    rev = "v${version}";
+    sha256 = "sha256-MixwIhyymruruV8G8PjmR9EoZBpaDVBCKBccSFL0lS8=";
+  };
+
+  propagatedBuildInputs = [ pkgs.ncurses ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  meta = {
+    description = "ncurses based snake game for the terminal";
+    homepage = "http://github.com/alexdantas/nSnake";
+    license = lib.licenses.gpl3Only;
```

Based on https://github.com/alexdantas/nSnake it looks like this should be:

```suggestion
    license = lib.licenses.gpl3Plus;
```

---

I haven't seen this previously, but when reading the actual license, you are right.